srcpkgs/python3-poppler-qt5/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'python3-poppler-qt5'
+pkgname=python3-poppler-qt5
+version=0.75.0
+revision=1
+wrksrc="python-poppler-qt5-${version}"
+build_style=python3-module
+hostmakedepends="python3-sip-devel pkg-config qt5-devel poppler-qt5-devel
+ python3-PyQt5-devel python3-devel"
+makedepends="python3-sip-devel pkg-config qt5-devel poppler-qt5-devel
+ python3-PyQt5-devel python3-devel"
+depends="poppler-qt5 python3-PyQt5"
+short_desc="Python bindings for libpoppler-qt5"
+maintainer="chili-b <dexter.gaonshatford@gmail.com>"
+license="LGPL-3.0-or-later"
+homepage="https://github.com/frescobaldi/python-poppler-qt5"
+changelog="https://github.com/frescobaldi/python-poppler-qt5/blob/v${version}/ChangeLog"
+distfiles="${PYPI_SITE}/p/python-poppler-qt5/python-poppler-qt5-${version}.tar.gz"
+checksum=ea0ec9ebe995705ab19a301290365652e62bab5c9b05db5697c7bf2173335107
+
+do_build() {
+	# Cludge to fix build under armv7l, armv6l
+	if [[ $XBPS_TARGET_MACHINE == arm* ]]; then
+		export CFLAGS="-mtune=generic-armv7-a "
+		export CFLAGS+="-I${XBPS_CROSS_BASE}/${py3_inc}"
+	fi
```

Comment:
If you need those CFLAGS,  you should set at least the `-I` one for all builds.

And I'd like to see a CI run with the mtune cludge removed.

Reply:
IIRC the CFLAGS are needed only for cross builds to ARM because their options aren't automatically configured correctly at build time. I will push a version with the mtune thing removed so you can see.